portality/datasets.py

```diff
@@ -1,570 +1,48 @@
-# -*- coding: UTF-8 -*-
-# the comment above is for the Python interpreter, there are Unicode
-# characters written straight into this source file
-
-import json
+import pycountry
```

Comment:
This file is way nicer than it was, though it still (necessarily) has a lot of the stuff that gets executed on import that I'm not that keen on.  I wonder if we could just tidy it up a little further by pulling together all the properties that you might want to use after import together at the top with a bit of documentation around them.  Even if we then go on to populate those objects further down the file.  Like:

```
## These are the data structures you might want to access...
# country options - [some documentation about that]
country_options = [('', '')]
# currency options - [some documentation about that]
currency_options = [(CURRENCY_DEFAULT, CURRENCY_DEFAULT)]

# ... etc
```